Library/OcPciIoLib/OcPciIoLib.c

```diff
@@ -0,0 +1,75 @@
+/** @file
+   PciIo overrides
+
+   Copyright (c) 2023, xCuri0. All rights reserved.<BR>
+   SPDX-License-Identifier: BSD-2-Clause-Patent
+
+ **/
+
+#include "OcPciIoU.h"
+#include <Library/OcPciIoLib.h>
+
+EFI_CPU_IO2_PROTOCOL *
+OcPciIoInstallProtocol (
+  IN BOOLEAN  Reinstall
+  )
+{
+  EFI_TPL                          Tpl;
+  UINTN                            Index;
+  EFI_STATUS                       Status;
+  EFI_CPU_IO2_PROTOCOL             *mCpuIo;
+  UINTN                            HandleCount;
+  EFI_HANDLE                       *HandleBuffer;
+  EFI_PCI_ROOT_BRIDGE_IO_PROTOCOL  *PciRootBridgeIo;
+
+  mCpuIo = InitializeCpuIo2 ();
```

Comment:
This is a local variable, not global variable. Please rename `mCpuIo` to `CpuIo` in this function.